src/Microsoft.Diagnostics.EventFlow.Inputs.Serilog/LoggerSinkConfigurationEventFlowExtensions.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.Diagnostics.EventFlow;
```

Comment:
If you do not mind, can you add a comment saying this file is licensed under MIT license?